topics/admin/slides/introduction.html

```diff
@@ -88,7 +88,6 @@
 
 ### Tutorials
 
-- [Move from dev instance to production instance](dev_to_production.html)
-- ['10 rules' for Setting up a Galaxy Instance as a Service](10rules_for_GaaS.html)
-- [Galaxy and Docker](galaxy_docker.html)
-- [Galaxy Database schema](database_schema.html)
+- [Deployment and Platform options](http://galaxyproject.github.io/training-material/topics/admin/tutorials/deployment-platforms-options/slides.html#1)
```

Comment:
Can you replace `http://galaxyproject.github.io/training-material` by `{{ site.baseurl }}`? Thanks